extension/background.js

```diff
@@ -11,8 +11,12 @@ chrome.commands.onCommand.addListener(function(command) {
 chrome.runtime.onMessage.addListener(
     function(request, sender, sendResponse) {
         console.log("Received tab message: ", request);
-        if(request.command == "activateTab" && sender.tab && activeTabs.indexOf(sender.tab.id) == -1){
-            activeTabs.push(sender.tab.id);
+        if(request.command == "activateTab" && sender.tab){
```

Comment:
I thought if was treated like a function and thus you don't (in good form) add spaces.